FireMotD

```diff
@@ -159,6 +159,22 @@ VersionComparison () {
   return 3
 }
 
+function IsInstalled {
+  if [[ -x "/usr/bin/yum" ]] ; then
+    YumExec=$(which yum 2>/dev/null)
+    if yum list installed "$@" >/dev/null 2>&1; then
+     true
```

Comment:
indention not correct

Reply:
ha yes I know the indention is different, but i switched from 4 spaces to 2 lately.. :)

Reply:
I saw that... but here it is 1 space.
Imho you should switch the whole script soon and use an editor supporting auto code formatting to bring everything in order.